### ISSUE TITLE
Timeline thumbnails issues

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#1271]
- [ ] overlay image popups too quick/or get in the way sometimes
- [ ] zoom is reset each time after we paint (e.g. on OS X we can change zoom with mouse wheel, after doing that, if we paint, the zoom is reset to the configured zoom level in preferences)
- [ ] vertical scroll jumps in big chunks when we have a big zoom (e.g. Zoom = 10)
- [ ] when we change zoom with mouse wheel we must update the scroll area

**Comment by dacap** [comment]
hi @zed9h, here I've just written down some tasks to improve the thumbnails, if you are going to take one of these items, please make a comment here 👍  (I'll do the same if I start working on one of these items)

**Comment by zed9h** [comment]
Cool, so, you liked this solution? 
It's nice but, I still miss the thumbs on the small timeline. 
Another idea: the overlay could be hidden by default when the timeline is larger.

This week were too busy, I will try to take a look at it in the next couple of weeks.

**Comment by dacap** [comment]
> Cool, so, you liked this solution? 

I liked the solution so far. Other possibility would be to move the thumbnail inside the cels-box, but I'm not 100% sure about this:

![screen shot 2016-10-03 at 11 49 04 am](https://cloud.githubusercontent.com/assets/39654/19041828/0ab1ded4-8960-11e6-9b4a-b7d101710e5e.png)

> It's nice but, I still miss the thumbs on the small timeline. 

Do you mean something like this:
![screen shot 2016-10-03 at 11 54 46 am](https://cloud.githubusercontent.com/assets/39654/19041862/34c26586-8960-11e6-8f40-f6dfb60228ce.png)

We can still have small thumbnails using the Force checkbox in the timeline options.

> Another idea: the overlay could be hidden by default when the timeline is larger.

I'll think more about this, I've to see how it works with different approaches (e.g. different positions for the overlay).

> This week were too busy, I will try to take a look at it in the next couple of weeks.

No problem. Just please make a comment if you start with some of the tasks, I'll do the same.

**Comment by zed9h** [comment]
> Do you mean something like this:
> ![image](https://cloud.githubusercontent.com/assets/7734/19042135/5015e7b2-8961-11e6-9096-8e79f9ddeb5e.png)

Yes, something like this, because the highest usefulness of the thumbnails is for large timelines, where there's also little space to spare.

> Other possibility would be to move the thumbnail inside the cels-box, but I'm not 100% sure about this:
> ![image](https://cloud.githubusercontent.com/assets/7734/19042082/0f146860-8961-11e6-8eef-0fb289a7db58.png)

This seems to be the very best solution! And could work with the small thumbs too.

What about test it, before the task "linked cels could be shown in the following way"?

If I start any of these tasks I tell you.

**Comment by dacap** [comment]
> What about test it, before the task "linked cels could be shown in the following way"?

Yes, I'll update the issue.

**Comment by zed9h** [comment]
Taking a look at this:
- vertical scroll jumps in big chunks when we have a big zoom (e.g. Zoom = 10)
- add zoom with ctrl+mouse wheel (using alt actually, ctrl is used)

**Comment by zed9h** [comment]
Doing that I also changed this:
-  zoom is reset each time after we paint (e.g. on OS X we can change zoom with mouse wheel, after doing that, if we paint, the zoom is reset to the configured zoom level in preferences)
- when we change zoom with mouse wheel we must update the scroll area

on the PR above.

**Comment by KasumiArai** [comment]
I've just seen a request for this:
![image](https://cloud.githubusercontent.com/assets/8582123/23340471/e9626cfe-fc04-11e6-972f-f17c184117a8.png)
The blended result of all layers for that frame, rather than cels. It could save visual space too, since it's one image regardless of layers.

**Comment by zed9h** [comment]
Nice idea, I will try to find some time to work on these items.

**Comment by dacap** [comment]
I'll be seeing some items of this issue because I'm trying to implement #1342

**Comment by oceanhahn** [comment]
Regarding "keyboard+mouse wheel combinations":

I would love to have the option (Preferences/Timeline) of swapping the MouseWheel (verticaly scroll/Layers) and SHIFT-MouseWheel (horizontal scroll/Frames) behaviour on MW over the Timeline. I tend to fiddle Frames more often than Layers, and being able to scroll more casually (which is, without SHIFT) would be nice.
